### ISSUE TITLE
`git-tag` task does not use provided tag name which causes the message to be used in its place instead

### ISSUE THREAD

**Issue by esuomi** [degree9/boot-semgit#10]
Relevant piece of code: [boot_semgit.clj#L204-L216](https://github.com/degree9/boot-semgit/blob/1ddae4f6a44cf76f9f836875a813f1f86dc8f5d4/src/degree9/boot_semgit.clj#L204-L216)

Furthermore annotated tag would be more useful instead of just plain tag, so I suspect the conjoining should add `-a` and `-m` as well so that the whole thing would look something like
```clojure
(cond-> ["tag"]
  (:delete *opts*) (conj "--delete")
  (:force *opts*)  (conj "--force")
  name             (conj "-a" name)
  message          (conj "-m" message))]
```

**Comment by burn2delete** [comment]
Thanks for reporting! This looks like a bad copy-paste when initially building the tasks.

**Comment by burn2delete** [comment]
Fixed in #11

**Comment by burn2delete** [comment]
Please try `1.3.0-SNAPSHOT`
